### PR TITLE
modified navbar and footer

### PR DESCRIPTION
**Pull request by yv2003**
made changes in navbar and footer

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -1,96 +1,104 @@
...
+    setOpen((prev) => !prev);
+    console.log(open);
+  };
+  return (
+    <div className="bg-[#0C1030] font-sans">
+      <div className="mx-auto  px-4 sm:px-6 lg:px-8">
+        <div className="flex items-center justify-between h-16 pl-4">
+          {/* Logo */}
+          <div
+            class="flex items-center gap-3 rounded-full "
```
Comment (JayeshVP24) [inline src/components/Navbar.jsx:34]: figure out gradient border

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

File: src/components/Navbar.jsx (hunk 1 above)
Comment (yv2003) [reply src/components/Navbar.jsx:34]: i have added border but the background and the border keep overlapping even after adding overflow hidden attribute

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -1,96 +1,105 @@
-import React, { useState } from 'react';
-import { FaBars, FaTimes } from 'react-icons/fa';
+import React, { useState } from "react";
+import { FaBars, FaTimes } from "react-icons/fa";
 
 const navlinks = [
```
Comment (JayeshVP24) [inline src/components/Navbar.jsx:4]: home
about us -> href: /#about
suggestions
template
features

change navbar content to above

File: src/components/Navbar.jsx [hunk 3]
```diff
@@ -1,96 +1,105 @@
...
+          {/* Logo */}
+          <div
+            class="flex items-center gap-3 rounded-full border-2 border-white border-opacity-25"
+            style={{
+              background:
+                "linear-gradient(218.92deg, rgba(255, 255, 255, 0.25) 30.2%, rgba(255, 255, 255, 0) 105.59%);",
+              backdrop: "blur(4px);",
+              overflow:"hidden;",
+            }}>
+            <img src="/Footer/logo.png" alt="" class="h-[49px] pl-1" />
```
Comment (JayeshVP24) [inline src/components/Navbar.jsx:41]: remove `pl-1` add `p-1`

File: src/components/Footer/index.astro [hunk 4]
```diff
@@ -1,11 +1,12 @@
-<div class="sm:h-[210px] h-full py-6 px-5 sm:py-[48px] sm:px-[41px] border-t bg-[#101828] flex   font-sans gap-5 flex-col">
+<div class="sm:h-[210px] h-full py-6 px-5 sm:py-[48px] sm:px-[41px] border-t bg-[#0C1030] flex   font-sans gap-5 flex-col">
 
     <!-- top -->
     <div class=" w-full min-h-fit  flex flex-col sm:flex-row  justify-between items-center ">
```
Comment (JayeshVP24) [inline src/components/Footer/index.astro:4]: add `gap-4`

File: src/components/Footer/index.astro
Comment (JayeshVP24) [inline src/components/Footer/index.astro:1]: change linux@tcetmumbai.in -> opensource@tcetmumbai.in at line 27

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

**Review by JayeshVP24 — APPROVED** [review]
LGTM

**Pull request merged**